packages/connect-core/src/entities/Organization.ts

```diff
@@ -105,6 +105,11 @@ export default class Organization {
     return this.#address || '' // The || '' should never happen but TypeScript requires it.
   }
 
+  get provider(): ethers.providers.Provider {
```

Comment:
May be useful to have access to the same provider the connect library initialize.

Reply:
Good point yes!

I think we could do this for now, and when making the “connection context” changes, move this inside a single `.connection` property that would contain the provider, the connector, etc. That way, we wouldn’t add too many properties on `Organization` that are in fact related to the connection.